packages/expo-constants/android/src/main/java/expo/modules/constants/ConstantsService.java

```diff
@@ -159,4 +165,15 @@ private static long getLongVersionCode(PackageInfo info) {
     }
     return info.versionCode;
   }
+
+  private @Nullable String getAppConfig() {
+    try (InputStream stream = mContext.getAssets().open(CONFIG_FILE_NAME)) {
+      return IOUtils.toString(stream, StandardCharsets.UTF_8);
+    } catch (FileNotFoundException e) {
+      // do nothing, expected in managed apps
```

Comment:
will this be called at all in managed apps? looking at [ConstantsBinding.java](https://github.com/expo/expo/blob/master/android/expoview/src/main/java/versioned/host/exp/exponent/modules/universal/ConstantsBinding.java) we override `getConstants()`. so this could be an opportunity for us to log that the app config file wasn't found

Reply:
Good point. Will make that change